app/src/main/java/com/amaze/filemanager/ui/dialogs/GeneralDialogCreation.java

```diff
@@ -1034,6 +1035,15 @@ public static void showPasswordDialog(
     textfield.setHint(promptText);
     textfield.setInputType(InputType.TYPE_CLASS_TEXT | InputType.TYPE_TEXT_VARIATION_PASSWORD);
 
+    textfield.requestFocus();
```

Comment:
Please make it an extension function for textfields and replace all code duplicates with the extension.